src/content/helpers/faceit-api.js

```diff
@@ -135,3 +135,6 @@ export const getPlayerHistory = async (userId, page = 0) => {
     `/match-history/v5/players/${userId}/history/?from=${from}&to=${to}&page=${page}&size=${size}&offset=${offset}`
   )
 }
+
+export const getMatchmakingQueue = queueId =>
+  fetchApiMemoized(`/queue/v1/queue/matchmaking/${queueId}`)
```

Comment:
This doesn't seem to work for hub games, which you can see by either going to [my profile](https://www.faceit.com/en/players/poacher2k/stats/csgo) or a player like [rain's profile](https://www.faceit.com/en/players/rain/stats/csgo)

Reply:
This is correct. Endpoints for matchmaking queues and hub queues seem to have been separated by FACEIT.

Reply:
I wasn't aware of that! it's been a while since I played, and back then it did affect ELO. thanks for clearing it up!